static/src/javascripts/projects/common/modules/commercial/creatives/expandable-v2.js

```diff
@@ -39,18 +41,70 @@ define([
         }
     };
 
-    ExpandableV2.prototype.listener = function () {
-        if ((window.pageYOffset + bonzo.viewport().height) > (this.$adSlot.offset().top + this.openedHeight)) {
-            // expires in 1 week
-            var week = 1000 * 60 * 60 * 24 * 7;
-
-            storage.local.set('gu.commercial.expandable.' + this.params.ecid, true, { expires: Date.now() + week });
-            this.$button.toggleClass('button-spin');
-            $('.ad-exp__open-chevron').toggleClass('chevron-down');
-            this.$ad.css('height', this.openedHeight);
-            this.isClosed = false;
-            return true;
+    /**
+     * TODO: rather blunt instrument this, due to the fact *most* mobile devices don't have a fixed
+     * background-attachment - need to make this more granular
+     */
+    ExpandableV2.hasScrollEnabled = !detect.isIOS() && !detect.isAndroid();
```

Comment:
This is the same functionality as the scrollable MPU, we could look at updating it all at the same time at some point?